src/main/java/org/openstreetmap/atlas/geography/atlas/multi/MultiArea.java

```diff
@@ -1,12 +1,15 @@
 package org.openstreetmap.atlas.geography.atlas.multi;
 
+import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
 
 import org.openstreetmap.atlas.geography.Polygon;
 import org.openstreetmap.atlas.geography.atlas.items.Area;
 import org.openstreetmap.atlas.geography.atlas.items.Relation;
 
+import com.google.common.collect.Sets;
```

Comment:
Looks like we need a core `Sets` implementation :)

Reply:
Lol